types/angular-block-ui/angular-block-ui-tests.ts

```diff
@@ -1,3 +1,6 @@
+// TypeScript Version: 2.3
```

Comment:
from looking at the Travis error log, it seems that the version annotation goes only in index.d.ts